common/src/main/java/tds/common/logging/EventLogger.java

```diff
@@ -19,6 +19,12 @@
  * Centralized event logger.
  */
 public class EventLogger {
+
+    public enum LogEvent {
+        APP_STARTUP,
```

Comment:
This isn't a comment about this review per se, but I've seen this same enum (not just the class but the exact values) in multiple classes.  It would be nice if this `EventLogger` class was more centralized for the common events that are captured in all applications.  I'm still unclear how all this wires together outside of reviews so maybe that isn't possible with the current implementation.